apps/explorer/lib/explorer/chain.ex

```diff
@@ -3901,7 +3901,8 @@ defmodule Explorer.Chain do
           ""
       end
 
-    formatted_revert_reason = format_revert_reason_message(data)
+    formatted_revert_reason =
+      revert_reason |> format_revert_reason_message() |> (&if(String.valid?(&1), do: &1, else: revert_reason)).()
```

Comment:
this looks awkward and can be written using [then/2](https://medium.com/blackode/two-new-life-saving-functions-worth-knowing-elixir-1-12-tap-then-b0120fcfb93f) instead of an anonymous function